lib/build.rb

```diff
@@ -67,7 +67,7 @@ def query_branches
       branch_details = github_query(@client) { @client.branch(@repository, b.name) }
       skip_message_present = false
       begin
-        skip_message_present = branch_details.commit.message['[decent_ci_skip]']
+        skip_message_present = branch_details.commit.commit.message['[decent_ci_skip]']
```

Comment:
This is really the root of the changes here.  The commit message is nested deep inside another layer of hash.  Verified this works though.